v4/internal/expect.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package internal
```

Comment:
This file was copied from the `v3/internal/expect.go` file. I removed a couple of methods that had to do with connect replies and harvests. Otherwise, I changed nothing.